tokio-console/src/config.rs

```diff
@@ -95,6 +96,17 @@ pub enum OptionalCmd {
     ///     $ tokio-console gen-config > console.toml
     ///
     GenConfig,
+
+    /// Generate shell completions
+    ///
+    /// It should be saved in expected directory, depending on the shell used
+    ///
+    ///     $ tokio-console gen-completion zsh > $FPATH/_tokio_console
```

Comment:
since we know the shell from the command-line argument, should we just make the command install the completions to the correct location, instead?

Reply:
@hawkw Maybe it can install the completion but I think that It's better not to do it because user want to decide where is putted.

Reply:
hmm, isn't the completion location determined by the shell, rather than the user? i'm fine with the current approach, but think this is worth a potential follow-up.